libminifi/src/FlowController.cpp

```diff
@@ -302,6 +302,10 @@ void FlowController::load(const std::shared_ptr<core::ProcessGroup> &root, bool
       this->root_ = std::shared_ptr<core::ProcessGroup>(loadInitialFlow());
     }
 
+    if (root) {
+      root_->verify();
```

Comment:
shouldn't we check `root_`?

Reply:
Yes, we should. Big thanks for noticing the typo!